### PR TITLE
#1 - project setup

### LINKED ISSUE #1: Project Setup

### PR DESCRIPTION
**Pull request by agawle**
Will close #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^2.6.11",
```
Comment (krzysztofrewak) [inline package.json:12]: Why not Vue 3?

Comment (agawle) [reply package.json:12]: Buefy plugin that we are planning to use in this project does not support Vue 3 currently, so we sticked with Vue 2.

Comment (krzysztofrewak) [reply package.json:12]: So maybe it's a good moment to abandon this Buefy thingo. ;)

Comment (agawle) [reply package.json:12]: After some investigation, Buefy is nowhere near to be migrated to Vue 3. It seems to be a good idea to drop this package for now.

**Pull request merged**